app/src/main/java/com/android/streetworkapp/model/park/ParkNameRepository.kt

```diff
@@ -0,0 +1,11 @@
+package com.android.streetworkapp.model.park
+
+/** A repository interface for park names. */
+interface ParkNameRepository {
```

Comment:
Sonar raise an issue at this line: "[Make this interface functional or replace it with a function type.](https://sonarcloud.io/project/issues?pullRequest=91&open=AZMdlXonPZUF0SGa_CTm&id=SwEnt-Group8_Street-work-app)

Reply:
I will fix it, thank you!